tests/CMakeLists.txt

```diff
@@ -1,3 +1,39 @@
+if (DEFINED ENV{GTEST_VERSION})
+  set(GTEST_VERSION_STR $ENV{GTEST_VERSION})
+else()
+  unset(GTEST_CONFIG_EXECUTABLE CACHE)
+  find_program(GTEST_CONFIG_EXECUTABLE
+    NAMES "gtest-config" "gtest-conf")
```

Comment:
I don't see either of these programs on my system, (there's /usr/src/googletest/googletest/scripts/gtest-config.in but...). That implies this is an unreliable way to establish the version.

It is probably better to use `apt policy libgtest-dev` on Debian based systems, `rpm -q libgtest-dev` on RPM based systems, etc.